### PR TITLE
Task/79  resend activation email has been complete

### PR DESCRIPTION
**Pull request by jananikuruntham**

### CODE REVIEW

File: api/src/users/users.controller.ts [hunk 1]
```diff
@@ -204,6 +204,44 @@ export class UsersController {
     }
   }
 
+  @Post('sent-resend-link')
```
Comment (vsaravanakumar) [inline api/src/users/users.controller.ts:207]: Change the route name to send instead of sent

File: api/src/users/users.controller.ts [hunk 2]
```diff
@@ -204,6 +204,44 @@ export class UsersController {
...
+  @UsePipes(new ValidationPipe())
+  @HttpCode(HttpStatus.OK)
+  @UsePipes(new ValidationPipe())
+  async sentResendLinkToUser(
```
Comment (vsaravanakumar) [inline api/src/users/users.controller.ts:211]: Change the function name

File: api/src/users/users.controller.ts [hunk 3]
```diff
@@ -204,6 +204,44 @@ export class UsersController {
...
+    @Body('lastName') lastName: string,
+  ): Promise<{
+    message: string;
+    statusCode: number;
+  }> {
+    return this.userService.sentResendLinkToUser(email, firstName, lastName);
+  }
+
+  @Post('check-password')
+  async checkPassword(
```
Comment (vsaravanakumar) [inline api/src/users/users.controller.ts:223]: Change the function name to relevant

File: front/src/app/users/[id]/page.tsx [hunk 4]
```diff
@@ -94,6 +110,22 @@ const UserView = ({ params }: { params: IUserView }) => {
   >([]);
   const [confirmationOpen, setConfirmationOpen] = useState(false);
   const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
+  const [openModal, setOpenModal] = React.useState(false);
+  const [isVisible, setIsVisible] = useState<ICreatePasswordProps>({
+    showPassword: false,
+    showConfirmPassword: false,
+  });
+  const [confirmPassword, setConfirmPassword] = useState("");
```
Comment (vsaravanakumar) [inline front/src/app/users/[id]/page.tsx:118]: Remove the unused variables

File: api/src/users/users.controller.ts (hunk 1 above)
Comment (jananikuruntham) [reply api/src/users/users.controller.ts]: route name changed

File: api/src/users/users.controller.ts (hunk 2 above)
Comment (jananikuruntham) [reply api/src/users/users.controller.ts]: function name changed

File: api/src/users/users.controller.ts (hunk 3 above)
Comment (jananikuruntham) [reply api/src/users/users.controller.ts]: function name is changed

File: front/src/app/users/[id]/page.tsx (hunk 4 above)
Comment (jananikuruntham) [reply front/src/app/users/[id]/page.tsx:119]: unused variables are removed

**Pull request merged**